beetmoverscript/templates/maven_components.yml

```diff
@@ -0,0 +1,50 @@
+---
+metadata:
+    name: "Maven repository"
+    description: "Maps artifacts to spec'd maven location"
+    owner: "release@mozilla.com"
+
+s3_bucket_path: maven2/org/mozilla/components/{{ artifact_id }}/{{ version }}/
```

Comment:
This may suffer some changes as we might need to parametrize `{{ component }}` as well.

Reply:
We won't. We're migrating towards consistency across components in a way that they are all prefixed.

Reply:
\o/